### DISCUSSION TITLE
[Done] Подсветка строки при выборе (галочка)

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#290] [Есть идея, а что если...]
Мне кажется было бы удобно подсвечивать и всю строку по нажатию галочки, особенно удобно если широкие таблицы.
Как считаете?

![Без названия](https://user-images.githubusercontent.com/113771444/207610478-c3ba68f6-daa8-4d88-89b4-397fd51db038.jpg)

**Comment by ttm-alexeypolunin** [comment]
Да, надо добавить чуть отличающуюся

**Reply by ttm-alexeypolunin** [reply]
Готово, в течении пары дней будет в репо на GitHub
